### PR TITLE
Feature: Refresh icons

### PR DESCRIPTION
**Pull request by Shomnipotence**
image

### CODE REVIEW

**Comment by jarno9981** [comment]
Looks good but waiting for @IviriusMain the comment about it

File: Delivery/AppRT/Products/Associated/RunBox.ico
Comment (Lamparter) [inline Delivery/AppRT/Products/Associated/RunBox.ico:1]: Why did you delete the `.ico` files? They are used.

File: Delivery/AppRT/Products/Associated/rcontrol.ico
Comment (Lamparter) [inline Delivery/AppRT/Products/Associated/rcontrol.ico:1]: > Why did you delete the `.ico` files? They are used.

Same here.

**Review by Lamparter — CHANGES_REQUESTED** [review]

File: Delivery/AppRT/Products/Associated/rcontrol.ico
Comment (Shomnipotence) [reply Delivery/AppRT/Products/Associated/rcontrol.ico:1]: They are replaced by another ico file with the same name

File: Delivery/AppRT/Products/Associated/RunBox.ico
Comment (Shomnipotence) [reply Delivery/AppRT/Products/Associated/RunBox.ico:1]: They are replaced by another ico file with the same name

Comment (jarno9981) [reply Delivery/AppRT/Products/Associated/RunBox.ico:1]: No .ico files

Comment (Lamparter) [reply Delivery/AppRT/Products/Associated/RunBox.ico:1]: Oops! I didn't realise. Sorry!

**Comment by Lamparter** [comment]
NEVER merge without a PR approval.

**Pull request merged**